static/js/charts.js

```diff
@@ -0,0 +1,20 @@
+class ChartBuilder {
+	constructor(type) {
+		this.type = type;
+	}
+
+	init(data, className){
+		var fn = this[this.type + "Init"];
+		if(typeof fn === "function") fn(data, className);
```

Comment:
Just a remark: I'm fine with this `if` notation, but most teams here prefer a curly bracket notation.

Reply:
Amended